packages/actions/src/releasePackages/index.ts

```diff
@@ -0,0 +1,47 @@
+import { getInput, startGroup, endGroup, getBooleanInput, info } from '@actions/core';
+import { program } from 'commander';
+import { generateReleaseTree } from './generateReleaseTree.js';
+import { releasePackage } from './releasePackage.js';
+
+const excludeInput = getInput('exclude');
```

Comment:
Won't this also throw like devInput/dryRun, and thus would need a try catch>

Reply:
Only booleans throw. This returns an empty string when not present (and the Boolean input actually uses this internally)